aiosmtpd/docs/controller.rst

```diff
@@ -230,7 +232,24 @@ Controller API
       Start the server in the subthread.  The subthread is always a daemon
       thread (i.e. we always set ``thread.daemon=True``.  Exceptions can be
       raised if the server does not start within the *ready_timeout*, or if
-      any other exception occurs in while creating the server.
+      any other exception occurs in :meth:`factory` while creating the server.
+
+      .. important::
+
+         If :meth:`start` raises an Exception,
+         :class:`Controller` does not automatically perform cleanup,
+         to support deep inspection post-exception (if you wish to do so.)
+         Cleanup must still be performed manually by calling :meth:`stop`
```

Comment:
"Semantic Linefeeds", as suggested (and somewhat evangelized) by @brandon-rhodes , explained very well in [his blog post](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).